<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>235</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="customerList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>35</y>
     <width>101</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>65</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Check out Item:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="barcode">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="checkoutItems">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>95</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer List</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="idText">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>15</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pebbleText">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>15</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pebble:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>230</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
